_posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md

```diff
@@ -21,7 +21,186 @@ sidebar:
 
 ## <i class="fa fa-ship" aria-hidden="true"></i> Homework 1
 
-This page is under construction. Please check back as we will be posting the first homework content soon.
+This assignment has two components:
+
+1. Part 1 - <a href="https://canvas.colorado.edu/courses/17063/quizzes/38134" target = "_blank">Homework 1 Quiz on CANVAS</a>. You need to log into CANVAS to complete this quiz. 
+
+2. Part 2 - `Jupyter Notebook` for Homework 1 (`ea-bootcamp-hw-1.ipynb`). You will need follow the instructions below **Submit Part 2 - Homework 1 Using Git/Github.com** to submit this component.
+
+You need to **complete both components of this assignment (Homework 1) by Friday, August 10th at 8:00 AM (U.S. Mountain Daylight Time)**. See <a href="https://www.timeanddate.com/worldclock/fixedtime.html?iso=20180810T08&p1=1243" target="_blank">this link</a>  to convert the due date/time to your local time. Each component contributes 50% to your final grade for Homework 1. 
+
+
+## <i class="fa fa-check-square-o fa-2" aria-hidden="true"></i> What You Need
+
+Be sure that you have completed all of the lessons from Days 1, 2, and 3 for the Earth Analytics Bootcamp. Completing the challenges at the end of the lessons will also help you with this assignment. 
+
+If you have not already forked and cloned the repository for homework 1, complete the following instructions:
+
+1. On Day 1 of the course, you will receive an email invitation to Homework 1 via `Github.com`.
+
+2. Accept the invitation, which will create a new private repository that includes your `Github.com` username in the name of the repository: `https://github.com/earthlab-education/ea-bootcamp-hw-1-yourusername`.
+
+3. `Fork` this repository to your `GitHub account`. Your forked repository will be available on `https://github.com/yourusername/ea-bootcamp-hw-1-yourusername`.
+
+4. Clone your forked repository `ea-bootcamp-hw-1-yourusername` to the `earth-analytics-bootcamp` directory on your computer (hint: `cd`, `git clone`).
+
+This repository will contain the the `Jupyter Notebook` for Homework 1 (`ea-bootcamp-hw-1.ipynb`).
 
 </div>
 
+
+## Part 1 - Homework 1
+
+This component will test your `Shell`, `Jupyter Notebook` and `Git` skills from Days 1 to 3. 
+
+Complete the <a href="https://canvas.colorado.edu/courses/17063/quizzes/38134" target = "_blank">Homework 1 Quiz on CANVAS.</a> You need to log into CANVAS to see this quiz. This component contributes 50 points to your homework grade (out of a total possible points of 100).
+
+
+## Part 2 - Homework 1
```

Comment:
```
## Part 2 - Complete and Submit a Jupyter Notebook
The second part of this homework assignment will involve you working on a set of questions in a jupyter notebook. Open the file `name-here` in your xxxx repo. Then  do the following:
```
Be super clear about the steps and what they are doign. when i read this i'm not sure if i'm opening this from the repo i forked above??